arangod/Pregel/Conductor/Conductor.cpp

```diff
@@ -349,18 +350,22 @@ auto Conductor::_initializeWorkers(VPackSlice additional) -> GraphLoadedFuture {
     TRI_ASSERT(vertexMap.size() == 1);
     auto [server, _] = *vertexMap.begin();
     _dbServers.push_back(server);
-    workers.emplace(server,
-                    std::make_unique<conductor::SingleServerWorkerApi>(
-                        _executionNumber, _feature, _vocbaseGuard.database()));
+    workers.emplace(
+        server, conductor::WorkerApi{server, _executionNumber,
+                                     std::make_unique<DirectConnection>(
+                                         _feature, _vocbaseGuard.database())});
   } else {
     for (auto const& [server, _] : vertexMap) {
       _dbServers.push_back(server);
-      workers.emplace(
-          server,
-          std::make_unique<conductor::ClusterWorkerApi>(
-              _executionNumber,
-              Connection::create(server, Utils::baseUrl(Utils::workerPrefix),
-                                 _vocbaseGuard.database())));
+      network::RequestOptions reqOpts;
+      reqOpts.timeout = network::Timeout(5.0 * 60.0);
```

Comment:
For what is this a timeout? Why is it 5*60?

Reply:
It was defined in Conductor::_sendToAllDBServers, which was used before to send the messages. I did not want to change the connection options, so I copied it here. I guess it means 5 minutes. We need it because we send a REST request with retry (see NetworkConnection.cpp line 79): We retry sending the request until the timeout is exceeded.